graph2tac/tfgnn/tasks.py

```diff
@@ -1271,8 +1271,7 @@ def create_inference_model(self, tactic_expand_bound: int, graph_constants: Grap
 
         # [batch_size, ]
         no_local_context_proofstates = scalar_proofstate_graph.context['local_context_ids'].row_lengths() == 0
-        no_global_context_proofstates = tf.fill(dims=(proofstate_graph.total_num_components,),
-                                                value=len(graph_constants.global_context) == 0)
+        no_global_context_proofstates = scalar_proofstate_graph.context['global_context_ids'].row_lengths() == 0
```

Comment:
Why do we have this?  When would it come up that there is literally nothing in the global context?  I think even tests have stuff in the global context, right?  But since it is in the code, we might as well just leave it for now.  Thanks for the fix.

Reply:
I don't know either, I agree it is probably a relict, I was just not fully sure